scripts/update-versions-for-release

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
-# **Run this script and commit changes BEFORE creating a new release tag in Github**
+# **Run this script and commit changes AFTER creating a local tag for the new release and BEFORE creating a remote tag for the new release**
+# Pushing the local tag to remote / creating a new remote tag on Github will trigger the release process.
```

Comment:
consider renaming `NEW_VERSION` and `OLD_VERSION` below to match with Latest/Previous

Reply:
good point 👍